plugins/org.eclipse.fordiac.ide.library/src/org/eclipse/fordiac/ide/library/LibraryManager.java

```diff
@@ -764,6 +768,9 @@ public void resolveDependencies(final IProject project, final TypeLibrary typeLi
 
 		checkLibChanges();
 
+		// remove when no longer needed
+		changeLibFolders(project, typeLibrary);
```

Comment:
Please rename.
i had to think a long time until i recognised what this method should do
maybe  **moveLinksToVirtualFolder** would be better